metricbeat/module/coredns/stats/_meta/fields.yml

```diff
@@ -0,0 +1,90 @@
+- name: stats
+  type: group
+  description: >
+    Contains statistics related to the coreDNS service
+  release: beta
+  fields:
+    - name: panic.count.total
+      type: long
+      description: >
+         Total number of panics
+    - name: dns.request.count.total
+      type: long
+      description: >
+         Total query count
+    - name: dns.request.duration.sec.bucket.*
```

Comment:
We only use  `sec` if the unit is actually seconds. It seems here and below it's in nano seconds. Based on (https://www.elastic.co/guide/en/beats/devguide/current/event-conventions.html) we should use `ns`, but for this case I could also see that we skip the unit part and only have `dns.request.duration.bucket.*`. Same below.

Reply:
:+1: